interactions/client/get.pyi

```diff
@@ -0,0 +1,116 @@
+from typing import overload, Type, TypeVar, List, Iterable, Optional, Callable, Awaitable, Literal
+
+from interactions.client.bot import Client
+from enum import Enum
+from ..api.models.channel import Channel
+from ..api.models.guild import Guild
+from ..api.models.member import Member
+from ..api.models.message import Message, Emoji, Sticker
+from ..api.models.user import User
+from ..api.models.webhook import Webhook
+from ..api.models.role import Role
+
+_T = TypeVar("_T", Channel, Guild, Webhook, User, Sticker)
```

Comment:
`_T` is really meant for allowing any typing under the type variable. I believe you're wanting to use `_A` for this manner which you're not following.